HPE3PARPSToolkit-CLI.psm1

```diff
@@ -17604,10 +17628,10 @@ Function Find-3parCage
 		}	
 	}	
 	$Result = Invoke-3parCLICmd -Connection $SANConnection -cmds  $cmd	
-	write-debuglog "  executing Find-3parCage Command , surface scans or diagnostics on physical disks with the command --> $cmd " "INFO:" 	
+	write-debuglog "  executing Find-3parCage Command , surface scans or diagnostics on physical disks with the command   " "INFO:" 	
 	if([string]::IsNullOrEmpty($Result))
 	{
-		return  "SUCCESS : Find-3parCage Command Executed Sucessfully $Result"
+		return  "SUCCESS : Find-3parCage Command Executed Successfully $Result"
```

Comment:
In the most of the return messages, each word marked s capital letter. Not required this. Please re-check all the return strings.
So, here the return sentences as can "Success : Find-3parCage cmdlet executed successfully $Result"

Reply:
Updated.